### PR TITLE
Untar to the current working directory.

### PR DESCRIPTION
**Pull request by kunday**
Avoid extra cp & rm.

/cc @seancaffery @wushujames

### References
 - https://github.com/zendesk/maxwell/pull/147

### Risks
 - None

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,8 +5,8 @@ RUN apt-get update && apt-get install -y openjdk-7-jre curl
 RUN mkdir /app
 WORKDIR /app
 
-RUN curl -sLo - https://github.com/zendesk/maxwell/releases/download/v0.14.2/maxwell-0.14.2.tar.gz | tar zxvf -
-RUN cp -r maxwell-*/* . && rm -rf maxwell-*
+RUN curl -sLo - https://github.com/zendesk/maxwell/releases/download/v0.13.1/maxwell-0.13.1.tar.gz \
```
Comment (token-cjg) [inline Dockerfile]: Is there any particular reason the release tag is being changed here from 0.14.2 to 0.13.1?

Comment (kunday) [inline Dockerfile]: Good point, I guess my auto merge downgraded it :(

**Comment by seancaffery** [comment]
:+1:

**Comment by fandoweb** [comment]
:+1:

**Pull request merged**